### PR TITLE
Theming

### PR DESCRIPTION
**Pull request by 31453**
- improve dark theme colors
- apply themes to all buttons
- apply themes to alert/info areas
- add arkime light/dark themes (arkime light is new default)
- allow user to select logo

---
Arkime Light Theme
Screen Shot 2020-11-16 at 10 16 54 AM
---
Arkime Dark Theme
Screen Shot 2020-11-16 at 10 18 49 AM
---
Select Logo
Screen Shot 2020-11-16 at 12 39 28 PM
---

### CODE REVIEW

**Comment by BryceV** [comment]
- "map land" + "map water" could be under a "Sessions Map" header
- "Source Packets" + "Destination Packets" could be under a "SPI Visualization Graph" header. Is there a color value we can set for normal bar graphs?

File: viewer/vueapp/src/components/utils/Navbar.vue [hunk 1]
```diff
@@ -104,6 +104,7 @@ import ESHealth from './ESHealth';
 export default {
   name: 'MolochNavbar',
   components: { ESHealth },
+  props: [ 'settings' ],
```
Comment (BryceV) [inline viewer/vueapp/src/components/utils/Navbar.vue:107]: Does navbar not have access to the css vars like the other components?

File: viewer/vueapp/src/components/settings/Settings.vue [hunk 2]
```diff
@@ -1230,6 +1230,33 @@
...
+          <hr>
+          <h3>Logos</h3>
+          <p>
+            Pick from these logos
+          </p>
+          <div class="row well logo-well mr-1 ml-1">
+            <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12 mb-2 mt-2 logos"
+              v-for="logo in logos"
+              :key="logo.location">
+              <img :src="logo.location" alt="hoot" />
```
Comment (BryceV) [inline viewer/vueapp/src/components/settings/Settings.vue:1243]: logo.name for alt?

File: viewer/vueapp/src/App.vue [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <template>
   <div v-if="compatibleBrowser">
-    <moloch-navbar></moloch-navbar>
+    <moloch-navbar :settings="user.settings">
```
Comment (BryceV) [inline viewer/vueapp/src/App.vue:3]: Why do we pass it from here instead of getting from store like the other components?

Comment (31453) [reply viewer/vueapp/src/App.vue:3]: Good eye! User is already in the navbar component 👍

**Pull request merged**